nucypher/cli/commands/alice.py

```diff
@@ -349,15 +375,14 @@ def grant(general_config,
           bob_encrypting_key, bob_verifying_key, label,
 
           # Other
-          m, n, expiration, value,
+          m, n, expiration, value, rate,
```

Comment:
How does this work if you provide both?

Reply:
It fails in `BlockchainPolicy`. See https://github.com/nucypher/nucypher/pull/1602#discussion_r369300611